### PR TITLE
Local web3 node

### PR DESCRIPTION
**Pull request by StanislavBreadless**

### CODE REVIEW

File: docker-compose-zksync-node.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    depends_on:
+      - postgres
+    init: true
+    env_file: data-restore.env
+    volumes:
+    - type: bind
+      source: ./volumes/data-restore
+      target: /pg_restore
+  server:
+    image: "matterlabs/server:ca06de20e"
```
Comment (Deniallugo) [inline docker-compose-zksync-node.yml:24]: let's do latest

File: data-restore.env [hunk 2]
```diff
@@ -15,4 +15,24 @@ PG_DUMP=
...
+DATABASE_URL=postgres://postgres@postgres:5432/plasma
+# The next three parameters aren't relevant for the local node, but needed to be supplied anyway
+DATABASE_POOL_SIZE=10
+DATABASE_REJECTED_TRANSACTIONS_MAX_AGE=336
+DATABASE_REJECTED_TRANSACTIONS_CLEANER_INTERVAL=24
+
+# The port to which the zksync node will bind to
+API_WEB3_PORT=3002
+# Max difference between blocks in `eth_getLogs` method.
+API_WEB3_MAX_BLOCK_RANGE=10
```
Comment (Deniallugo) [inline data-restore.env:32]: It's safe to make this number bigger, especially for local usage

Comment (StanislavBreadless) [reply data-restore.env:32]: I will leave a comment about it

File: docker-compose-zksync-node.yml (hunk 1 above)
Comment (StanislavBreadless) [reply docker-compose-zksync-node.yml:24]: Resolved in DM

**Review by Deniallugo — APPROVED** [review]

**Pull request merged**